### ISSUE TITLE
OnionShare 2.6 (Flatpak) Crashes on Startup under Debian 11 After First Reboot

### ISSUE THREAD

**Issue by georgemaschke** [onionshare/onionshare#1630]
On a new, fully patched instance of Debian 11:
1) Install and configure the OnionShare 2.6 Flatpak package (it runs as expected);
2) Reboot the computer;
3) OnionShare 2.6 consistently crashes on startup.

**Comment by mig5** [comment]
When you say it 'crashes', is there a stack trace? If you run the flatpak from the command line (perhaps with `-v` appended to the argument) is there a trace of the error?

Can you also check if there's a 'lock' file present in your config directory, which is a known issue that can prevent OnionShare opening the second time if that lock file is somehow still present: https://github.com/onionshare/onionshare/issues/1374

On Flatpak, this may not be in `~/.config/onionshare` but in `~/.var/app/org.onionshare.OnionShare/config/onionshare`

**Comment by georgemaschke** [comment]
As I didn't have the command line version installed, I first checked for a lock file. It was in the location you suggested for Flatpak. After deleting the lockfile, I can now run OnionShare 2.6, and it has continued to work after rebooting the computer. Thank you so much for your assistance in resolving this!

**Comment by georgemaschke** [comment]
Yes, OnionShare was still running when the computer shut down in a sudden and unexpected way. (I would normally manually shut down OnionShare before a system shutdown.)
